avocado/utils/disk.py

```diff
@@ -29,6 +29,12 @@
 from . import process
 
 
+class DiskUtilsError(Exception):
```

Comment:
I get your goal here, but it's usually a good idea for simple additions (such as a variable or function) to be introduced alongside its user.

Unless you're introducing an API that is supposed to be used by the end user, and not our own code, it's usually better to add it with the consuming code.  One other example is that it make things easier even for reviews.

So my suggestion is that you squash this into another commit.

Reply:
i defined as  i need to raise exception  in  `get_partition_info` sub routine and why i introduce this in another commit as it is kind of place holder and can be used  for any function on disk utility , if still think i need to squash in consumer commit i can do so please suggest